test/lint/lint-spelling.sh

```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/env bash
+#
+# Copyright (c) 2018 The Bitcoin Core developers
+# Distributed under the MIT software license, see the accompanying
+# file COPYING or http://www.opensource.org/licenses/mit-license.php.
+#
+# Check for spelling errors.
+
+export LC_ALL=C
+
+EXIT_CODE=0
+EXCLUDE_REGEXP="(src/example.cpp:.*example-false-positive-word) "
+SPELLING_ERRORS=$(codespell -d -q 3 $(git ls-files -- ":(exclude)src/leveldb/" ":(exclude)src/secp256k1/" ":(exclude)src/univalue/" ":(exclude)depends/" ":(exclude)doc/release-notes/" ":(exclude)src/qt/locale/" ":(exclude)test/functional/test_framework/__init__.py" ":(exclude)test/lint/lint-spelling.sh") | grep -vE "${EXCLUDE_REGEXP}")
```

Comment:
nit: using the long form options here would be more self-documenting: `--disable-colors --quiet-level=`

Reply:
Good point! Now fixed.